docs: remove Atom as a recommended code editor (#49974)
freeCodeCamp/freeCodeCamp: docs/how-to-setup-freecodecamp-locally.md

2. (Optional but recommended) [Set up an SSH Key](https://help.github.com/articles/generating-an-ssh-key/) for GitHub.

> 3. Install a code editor of your choice, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.

Reviewer: ```suggestion
3. Install a code editor of your choice. We recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```

My turn to nitpick - this is a comma splice (:

> 3. Install a code editor of your choice, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.

Author: ```suggestion
3. Install a code editor of your choice. Nevertheless, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```
How about this?

Reviewer: `Nevertheless` is just added fluff here, IMHO.

Author: We initially say 'You can install the code editor of your choice', but then we go very (IMO) cut and dry with our suggestion. Not a big deal of course - we can leave it as how Noami suggested. 

Maybe there is a less 'fluffy' word we can use to transition to our recommendation?

Reviewer: ```suggestion
3. Install a code editor of your choice. If you aren't sure which one to use, we recommend [Visual Studio Code](https://code.visualstudio.com/) — it's free and open source.
```

If we want a transition...

> 3. Install a code editor of your choice.

Reviewer 2: ```suggestion
```

>    Use your favorite text editor, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.

Reviewer 2: ```suggestion
3. Install a code editor of your choice, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```
